api/tests/unit/domain/services/mail-service_test.js

```diff
@@ -204,155 +206,139 @@ describe('Unit | Service | MailService', () => {
     const organizationName = 'Organization Name';
     const organizationInvitationId = 1;
     const code = 'ABCDEFGH01';
+    const translationsMapping = organizationInvitationTranslationsMapping;
 
-    let organizationInvitationEmailFr;
-    let organizationInvitationEmailEn;
+    it('should call sendEmail with from, to, organizationName', async () => {
 
-    let expectedOptions;
-
-    beforeEach(() => {
-      organizationInvitationEmailFr = translatedTextsFr['organization-invitation-email'];
-      organizationInvitationEmailEn = translatedTextsEn['organization-invitation-email'];
-
-      translatedTextsFr['organization-invitation-email'] = { title: 'Un titre' };
-      translatedTextsEn['organization-invitation-email'] = { title: 'A title' };
+      const locale = 'fr';
```

Comment:
Ici, une locale n'est pas nécéssaire.

Reply:
:thinking: je ne savais pas, et en regardant il y a une locale par défaut utilisée dans le cas.
J'ai donc:
- alimentée la locale à `undefined` ici pour indiquer que c'est un choix délibéré
- ajout un cas de test dans le context `according to locale`

Reply:
Quelque soit ton choix, il sera bon :)
J'ai déjà donné ma coche verte :)